### PR TITLE
Fix multiline comments highlighting in c++

### PR DESCRIPTION
**Pull request by rafalk342**
I've corrected handling multiline comments in c++. 
Every tag with a multiline comment is changed into a multiple tags with only one line.

### CODE REVIEW

**Comment by wcoder** [comment]
@rafalk342 Thanks for your help! #21 
I will check your solution for other languages.

File: src/highlightjs-line-numbers.js [hunk 1]
```diff
@@ -105,6 +106,37 @@
...
+		}
+		return result;
+	}
+
+	function changeMultilineComments(element) {
+		var result = '';
+		var children = element.childNodes;
+		for(var child in children){
+			if(children.hasOwnProperty(child)) {
+				if(children[child].className === "hljs-comment"){
```
Comment (anlambert) [inline src/highlightjs-line-numbers.js]: Testing against "hljs-string" also fixes the highlighting of multi-line strings

**Comment by anlambert** [comment]
For your information, there is a similar issue regarding the highlighting of multi-line strings (for instance, Python ones). See comment in the PR diff for the fix.

**Comment by wcoder** [comment]
Hi everyone

I investigated this issue #21 and made a common solution.

Please, try it on the separate branch:

https://github.com/wcoder/highlightjs-line-numbers.js/blob/bugfixes/multiline-nodes/src/highlightjs-line-numbers.js

**Comment by anlambert** [comment]
Hi @wcoder,

I confirm this fixes the encountered issues.

Cheers!

**Comment by rafalk342** [comment]
Hello @wcoder,
I also confirm that this fixes the issue.
Thanks for moving with it forward.

**Comment by wcoder** [comment]
Thank you for feedbacks and check.

Released new version [2.2](https://github.com/wcoder/highlightjs-line-numbers.js/releases/tag/v.2.2.0) with this fix.

**Pull request closed without merging**